apps/quasar/app.json

```diff
@@ -0,0 +1,11 @@
+{
+  "appCategory": "Asset Management",
```

Comment:
"appCategory" must be one of [DeFi, Social, NFT, DAO, Tool, Liquid Staking]

Reply:
Sure. Will update it soon.

Reply:
Update to DeFi